code/modules/reagents/reagents/toxin.dm

```diff
@@ -654,6 +654,41 @@
 
 	return ..()
 
+/datum/reagent/toxin/xeno_lightsanguinal
+	name = "light Sanguinal"
+	description = "A weaker version of the toxin Sanguinal, but just as deadly."
+	reagent_state = LIQUID
+	color = "#f3491f"
+	custom_metabolism = 1
+	purge_list = list(/datum/reagent/hypervene, /datum/reagent/medicine/ryetalyn, /datum/reagent/medicine/dylovene)
```

Comment:
i don't think you understand what the purge list is. It's the list of reagent that will be purged by the sanguinal, not the opposite

Reply:
done